docs/releases/1.10.txt

```diff
@@ -250,6 +250,10 @@ Forms
 * Form and widget ``Media`` is now served using
   :mod:`django.contrib.staticfiles` if installed.
 
+* Form now has the attribute :attr:`~django.forms.Form.use_required_attribute`.
+  When set to ``True``, required form fields will have the HTML attribute
```

Comment:
"When set to True (the default), ...." (and chop the next sentence)
Also add a mention in the backwards-incompatible changes section. Can mention "<form novalidate>" .